contrib/format-spss/README.md

```diff
@@ -0,0 +1,87 @@
+# Format Plugin for SPSS (SAV) Files
+This format plugin enables Apache Drill to read and query Statistical Package for the Social Sciences (SPSS) (or Statistical Product and Service Solutions) data files. According
+ to Wikipedia: [1]
```

Comment:
Wikipedia may be made as a reference instead of putting the long at the end of the doc.

Reply:
Fixed